scripts/state-sync.sh

```diff
@@ -2,13 +2,13 @@
 # It is recommended to have bash scripting experience to understand what this script does for you
 # Special thanks to our community member and validator MZONDER
 # request
-SNAP_RPC="https://rpc.getbze.com" # d-01 mz
+SNAP_RPC="https://rpc.getbze.com:443"
```

Comment:
That's odd. It worked for me without the port. I forgot to say i tested it on my sentries.